corehq/apps/app_manager/models.py

```diff
@@ -5713,6 +5713,20 @@ def undo(self):
         app.save()
 
 
+class SQLGlobalAppConfig(models.Model):
+    choices = [(c, c) for c in ("on", "off", "forced")]
+
+    domain = models.CharField(max_length=255, null=False)
+    app_id = models.CharField(max_length=255, null=False)
+    app_prompt = models.CharField(max_length=32, null=True, choices=choices, default="off")
+    apk_prompt = models.CharField(max_length=32, null=True, choices=choices, default="off")
+    apk_version = models.CharField(max_length=32, null=True)
+    app_version = models.IntegerField(max_length=32, null=True)
```

Comment:
I think that `IntegerField` shouldn't need/be able to use `max_length`. 

Why do all of these need `null=True` columns? Shouldn't all have an `app_version`? It appears that the `prompt`s have defaults, so they shouldn't be ever be null. i'm less sure about `apk_version`, but assume that would have a reasonable default as well.

Reply:
Updated the integer and the *_prompt fields in e3b579fe155. Staging has docs where app_version and apk_version are blank. Prod does not. I'm inclined to allow the versions to be null rather than fiddle with the data on staging (even though it may well be "wrong").